cucumber-expressions/go/default_transformer_test.go

```diff
@@ -0,0 +1,62 @@
+package cucumberexpressions
+
+import (
+	"github.com/stretchr/testify/require"
+	"reflect"
+	"testing"
+)
+
+func TestConvert(t *testing.T) {
+	t.Run("SimpleDefaultTransformer converts to kind string", func(t *testing.T) {
+		typeOfString := reflect.TypeOf("string")
+		assertTransforms(t, "Barbara Liskov", "Barbara Liskov", typeOfString)
+	})
+
+	t.Run("SimpleDefaultTransformer converts to kind string", func(t *testing.T) {
```

Comment:
What's different/unique about this test versus the one on line 10 that creates the `typeOfString`?

Reply:
Ah I see. The one on line 10 uses `Type` this one uses `Kind`. But I didn't rename the test.

Reply:
We're using both `Type` and `Kind` so `cucumber-go` can use reflection to get `Type` directly from methods via reflect while we can define the transforms for the basic types as `Kind`. It is not possible in go to reference a `Type` directly outside of type assertions.